packages/jest-haste-map/src/index.js

```diff
@@ -39,6 +39,7 @@ type Options = {
   cacheDirectory?: string,
   console?: Console,
   extensions: Array<string>,
+  forceNodeFSApi?: boolean,
```

Comment:
I hate to be that guy, but I think we should call this option `forceNodeFilesystemAPI` or `forceNodeFSAPI` or `forceFSAPI` or similar. `Api` is not the proper abbreviation.